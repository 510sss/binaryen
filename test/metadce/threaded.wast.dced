(module
 (type $none_=>_none (func))
 (import "env" "js_func2" (func $js_func_2))
 (export "wasm_func1" (func $wasm_func_1))
 (func $wasm_func_1
  (call $js_func_2)
 )
)
