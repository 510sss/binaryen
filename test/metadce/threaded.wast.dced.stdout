unused: func$wasm_func_4$3
unused: outside_js_function1
unused: wasm_export4
